crowdfunding_compassion/forms/project_creation_form.py

```diff
@@ -8,16 +8,47 @@
 ##############################################################################
 import logging
 
-from odoo import models, _
+from odoo import models, fields, _
 
 _logger = logging.getLogger(__name__)
 
 
-class ProjectCreationForm(models.AbstractModel):
-    _name = "cms.form.crowdfunding.project"
-    _inherit = ["cms.form"]
-
+class ProjectCreationWizard(models.AbstractModel):
+    _name = "cms.form.crowdfunding.wizard"
+    _inherit = "cms.form.wizard"
     _form_model = "crowdfunding.project"
+    _form_extra_css_klass = "crowdfunding_project_creation_from"
+    _wiz_name = _name
+
+    def _wiz_base_url(self):
+        return "/projects/create"
+
+    def form_check_permission(self, raise_exception=True):
+        # no need for this
+        pass
+
+    def wiz_configure_steps(self):
+        return {
+            1: {"form_model": "cms.form.crowdfunding.project.step1"},
+            2: {"form_model": "cms.form.crowdfunding.project.step1"},
+            3: {"form_model": "cms.form.crowdfunding.project.step1"},
```

Comment:
Not sure but should't these be cms.form.crowdfunding.project.step2 and cms.form.crowdfunding.project.step3 ?

Reply:
Oh yes :)